CMakeLists.txt

```diff
@@ -59,7 +59,7 @@ endif()
 add_dependencies(uuid CoreFoundation)
 
 set(swift_optimization_flags)
-if(CMAKE_BUILD_TYPE MATCHES Release)
```

Comment:
Oh, interesting!  This makes a lot of sense.  Please change this to:

```
if(NOT CMAKE_BUILD_TYPE MATCHES Debug)
```

Please don't merge this, you will break the Windows build.